tests/fixtures/main.rs

```diff
@@ -0,0 +1,33 @@
+mod deserialize;
+
+use crate::deserialize::Fixture;
+use annotate_snippets::{Renderer, Snippet};
+use snapbox::data::DataFormat;
+use snapbox::Data;
+use std::error::Error;
+
+fn main() {
+    #[cfg(not(windows))]
```

Comment:
Why `not(windows)`?

Reply:
There are deserialization errors on Windows. This was a problem with the [previous fixture test](https://github.com/rust-lang/annotate-snippets-rs/blob/e9b445c3d43dc5d1673a89dcdab691187a6b9566/tests/fixtures_test.rs#L22).